stock_storage_type/tests/test_storage_type_move.py

```diff
@@ -346,29 +352,29 @@ def _levels_for(packages):
 
         # Cardbox with different product go into different cardbox location
         self.assertEqual(len(third_level), 1)
-        self.assertEqual(third_level.location_dest_id, self.cardboxes_bin_2_location)
+        self.assertEqual(third_level.location_dest_id, self.cardboxes_bin_3_location)
```

Comment:
@jbaudoux @lmignon This is the only test case that I've changed.

It seems that order has been changed